tests/resources/test_item.py

```diff
@@ -1417,8 +1417,9 @@ async def test_search_datetime_validation_errors(app_client):
         resp = await app_client.post("/search", json=body)
         assert resp.status_code == 400
 
-        resp = await app_client.get("/search?datetime={}".format(dt))
-        assert resp.status_code == 400
+        # ValueError: Invalid RFC3339 datetime. - not sure about this
+        # resp = await app_client.get("/search?datetime={}".format(dt))
+        # assert resp.status_code == 400
```

Comment:
It would be better to fix the code rather than remove the test, yes?

Reply:
Here we added the converter to the BaseSearchGetRequest but if I remove it I get 200 responses - https://github.com/stac-utils/stac-fastapi/blob/cae227840ee231ecfdb9b4ef1d187142dfe534ce/stac_fastapi/types/stac_fastapi/types/search.py#L105-L112

Reply:
I don't think is the best way to do this: https://github.com/stac-utils/stac-fastapi/pull/670

Reply:
@jonhealy1 are you planning to do a new release now that 670 is merged ?

Reply:
Haha yes - I was just working on it

Reply:
this is resolved ?